src/views/TaggingView.vue

```diff
@@ -320,133 +302,42 @@ export default Vue.extend({
         }
       }, 300);
     },
-    pickDevice() {
-      const devices = Object.keys(this.recordingsByDevice).map(Number);
-      // NOTE(jon): Just pick the first device, we'll rely on the backend to choose these for us.
-      this.currentDeviceId = devices[0];
-    },
-    pickRecording: async function() {
-      const recordings = this.recordingsByDevice[this.currentDeviceId];
-      if (recordings && recordings.length !== 0) {
-        // NOTE(jon): Just take the first recording, we'll rely on the backend to choose these for us.
-        this.currentRecording = recordings.splice(0, 1)[0];
-
-        // If we took the last recording for a device, remove the device,
-        // and pick a next device.
-        if (recordings.length === 0) {
-          Vue.delete(this.recordingsByDevice, this.currentDeviceId);
-          this.pickDevice();
-        }
-
-        const {
-          tracks,
-          recording
-        }: {
-          tracks: Track[];
-          recording: Recording;
-        } = await this.$store.dispatch(
-          "Video/GET_RECORDING",
-          this.currentRecording.id
-        );
-        if (!tracks || tracks.length === 0) {
-          this.pickRecording();
-        } else {
-          this.tracks = tracks;
-          this.currentRecordingData = recording;
-          // Get the first track without human tags
-          this.currentTrackIndex = this.orderedTracks.findIndex(
-            track => !this.trackIsAlreadyTaggedByHuman(track)
-          );
-        }
+    async getRecording(biasToDeviceId?: DeviceId): Promise<boolean> {
+      let recordingResponse;
+      if (biasToDeviceId !== undefined) {
```

Comment:
The sense here looks wrong. Shouldn't this be `if (biasToDeviceId === undefined) {`? That would explain why when trying out the feature it seemed to jump around a lot when tagging tracks without using the next recording button :)

Also, the `if` isn't even needed as `needsTag`'s argument is optional anyway.